css-two-column-layout/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+   <div class=" margin-top  header-marg text-align-center">
+    <h1 class=" biggest-text col-10">SCALE,SPEED,AGILITY</h1>
+    <p class="col-9"> We've decided thousands of man hours to making healthcare institutions in the
+      Helium <br> network run 10x more efficiently and deliver an uparalled quality of care.
+    </p>
+    </div>
+   <div class="row">
+    <div class="col-5">
```

Comment:
`col-5` what? Is this meant to signify `column-fifth`? I would err on the side of being more descriptive with your class names.